bdii.spec

```diff
@@ -122,6 +122,11 @@ fi
 
 %changelog
 
+* Wed Sep 23 2020 Baptiste Grenier <baptiste.grenier@egi.eu> - 5.2.26-1
+- Truncate LDIF password file before updating (Enol Fernández, Andrea Manzi)
```

Comment:
Seems you swapped the authors of these entries: truncate passwd is Petr, base64 is Andrea and myself

Reply:
Oops, updated